common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1864,35 +1871,27 @@ private void finalizePartition() throws IOException {
       metaFile.getChannel().truncate(metaFile.getPos());
     }
 
-    void closeAllFilesAndDeleteIfNeeded(boolean delete) {
+    private void deleteAllFiles() {
       try {
-        if (dataChannel.isOpen()) {
-          dataChannel.close();
-        }
-        if (delete) {
-          dataFile.delete();
-        }
-      } catch (IOException ioe) {
-        logger.warn("Error closing data channel for {} reduceId {}",
+        if (!dataFile.delete()) {
+          logger.warn("Error deleting data file for {} reduceId {}",
```

Comment:
Make deletion failed messages in this method a `INFO` at a minimum, deletion could have failed due to other concurrent deletions.

Reply:
update it to ```INFO```